### PR TITLE
docs: Document SQL action

### PR DESCRIPTION
**Pull request by bessonm**
preview : https://bessonm.github.io/actions/sql/

### CODE REVIEW

File: docs/actions/sql.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+* `affectedRows`: Returns the number of affected rows. This is useful for **INSERT**, **UPDATE** or **DELETE** statements  
+-> `${#records.affectedRows}`
+
+* `count()`: Returns the number of results from a **SELECT** statement.  
+-> `${#records.count()}`
+
+* `headers`: Returns the list of columns names (List< String >)  
+-> `${#records.headers}`
+
+* `columns`: Returns a list of [Columns](#column) (List< Column >).
```
Comment (bessonm) [inline docs/actions/sql.md:82]: I couldn't find how to properly escape `<>` in a bullet list :/
might needs a look in the python md/html generator if someone is up for a contribution there :)

**Review by nbrouand — CHANGES_REQUESTED** [review]

File: docs/actions/sql.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+This action enables you to send SQL requests to a database.
+
+# Configuration
+
+!!! important "Database URL"
```
[the remark this answers is not in the document]
Comment (bessonm) [reply docs/actions/sql.md:5]: done

**Comment by bessonm** [comment]
2 notes: 
* I choose to do a collapsible example for the configuration example
* I choose to put the Logging results admonition with the "inputs" tab

Tell me what you think ?

**Comment by nbrouand** [comment]
> 2 notes:
> 
>     * I choose to do a collapsible example for the configuration example
> 
>     * I choose to put the Logging results admonition with the "inputs" tab
> 
> 
> Tell me what you think ?
Sorry but i dont like these. I was going to make a comment about these before seeing your comments :)
I prefer to have the example opened by default.
I was confused when i click on output. I prefer to not hide blocks when cliking on output

**Comment by bessonm** [comment]
> 

All right, I wasn't sure so I asked.
I'll change that

**Review by nbrouand — APPROVED** [review]

**Pull request merged**